federated_aws_cli/role_picker.py

```diff
@@ -38,6 +39,27 @@ def get_aws_env_variables(credentials):
     return result
 
 
+def get_aws_shared_credentials(credentials, role_arn):
+    # come up with a profile name that won't break AWS
+    result = "[{}]\naws_access_key_id={}\naws_secret_access_key={}\nrole_arn={}\n".format(
+        re.sub("[^a-zA-Z0-9]", "", role_arn),
```

Comment:
I'd say it's benefit us to be more intentional here about profile names as these are names that users interact with. If we take a `role_arn`, for example `arn:aws:iam::123456789012:role/MyRole` and use this `re.sub` we'll end up with `arnawsiam123456789012roleMyRole` which feels long and a bit confusing.

What about something that takes the account alias (which we get when we fetch the group role map) and the role name with a delimiter (like `-`), so for the above if the alias for account `123456789012` was `infosec-dev` then the profile name would be `infosec-dev-MyRole`

I'm find with the current method for now if you'd prefer to open an issue and fix this after this PR.